rfcs/theme-tokens.md

```diff
@@ -0,0 +1,247 @@
+# RFC: Theme Tokens
```

Comment:
THANK YOU for writing this up. I'm really happy seeing more write ups like this.
👍 👍 👍